cookbooks/aws-parallelcluster-platform/resources/nvidia_driver/partial/_nvidia_driver_common.rb

```diff
@@ -113,7 +113,7 @@ def compiler_version
 end
 
 def nvidia_kernel_module
-  if node['cluster']['nvidia']['kernel_open'] == "false"
+  if ['off', 'false', '0', 'no', false, 0].include?(node['cluster']['nvidia']['kernel_open'])
```

Comment:
Interesting way to catch all the vaules.
We could remove '0' or 'off' options since we're not using them in any part of the code, but at the same time we can keep them, they are not a problem

Can you extend the `nvidia_river_spec` to cover these cases, adding more combinations in this test:
```
describe 'nvidia_driver:nvidia_kernel_module' do
  [%w(false kernel), %w(true kernel-open)].each do |kernel_open, kernel_module|
...
```